yarn-error.log

```diff
@@ -0,0 +1,7445 @@
+Arguments: 
```

Comment:
Should this file have been checked in?

Reply:
Oops! Will remove and add to `.gitignore`